docs/tutorial/5_downloading_data.md

```diff
@@ -30,20 +30,63 @@ On Linux you can do this using the `-restrict-paths` command line argument, or b
 On Windows and Mac OS this option is in the "Access" tab of the configuration options.
 Globus will only be able to transfer files onto your machine in the specified paths.
 
+### The Globus web app
```

Comment:
Do we actually want to include all this? Also you seem to be missing the images if we do.